spring-cloud-dataflow-docs/src/main/asciidoc/api-guide.adoc

```diff
@@ -2067,6 +2067,85 @@ include::{snippets}/task-executions-documentation/list-task-executions-by-name/c
 include::{snippets}/task-executions-documentation/list-task-executions-by-name/http-response.adoc[]
 
 
+[[api-guide-resources-task-thin-executions-list]]
+==== List All Task Thin Executions
+
+The task executions endpoint lets you list all task executions with only top-level data.
+The following topics provide more details:
+
+* <<api-guide-resources-task-thin-executions-list-request-structure>>
+* <<api-guide-resources-task-thin-executions-list-request-parameters>>
+* <<api-guide-resources-task-thin-executions-list-example-request>>
+* <<api-guide-resources-task-thin-executions-list-response-structure>>
+
+
+
+[[api-guide-resources-task-thin-executions-list-request-structure]]
+===== Request Structure
+
+include::{snippets}/task-executions-documentation/list-task-thin-executions/http-request.adoc[]
+
+
+
+[[api-guide-resources-task-thin-executions-list-request-parameters]]
+===== Request Parameters
+
+include::{snippets}/task-executions-documentation/list-task-thin-executions/request-parameters.adoc[]
+
+
+
+[[api-guide-resources-task-thin-executions-list-example-request]]
+===== Example Request
+
+include::{snippets}/task-executions-documentation/list-task-thin-executions/curl-request.adoc[]
+
+
+
+[[api-guide-resources-task-thin-executions-list-response-structure]]
+===== Response Structure
+
+include::{snippets}/task-executions-documentation/list-task-thin-executions/http-response.adoc[]
+
+
+
+[[api-guide-resources-task-thin-executions-list-by-name]]
+==== List All Task Thin Executions With a Specified Task Name
+
+The task thin executions endpoint lets you list task executions with a specified task name.
+The following topics provide more details:
+
+* <<api-guide-resources-task-thin-executions-list-by-name-request-structure>>
+* <<api-guide-resources-task-thin-executions-list-by-name-request-parameters>>
+* <<api-guide-resources-task-thin-executions-list-by-name-example-request>>
+* <<api-guide-resources-task-thin-executions-list-by-name-response-structure>>
+
+
+
+[[api-guide-resources-task-thin-executions-list-by-name-request-structure]]
+===== Request Structure
+
+include::{snippets}/task-executions-documentation/list-task-thin-executions-by-name/http-request.adoc[]
+
+
+
+[[api-guide-resources-task-thin-executions-list-by-name-request-parameters]]
```

Comment:
Can you verify that the request parameters are being generated properly.  I'm seeing, 
```
Unresolved directive in api-guide.adoc - include::/Users/grenfro/project/spring-cloud-dataflow/spring-cloud-dataflow-docs/../spring-cloud-dataflow-classic-docs/target/generated-snippets/task-executions-documentation/list-task-thin-executions-by-name/request-parameters.adoc[]
```

Reply:
![image](https://github.com/user-attachments/assets/01179771-7ded-42bc-a855-c2dbedd3086f)

Reply:
Found the issue that caused this.   Created https://github.com/spring-cloud/spring-cloud-dataflow/issues/6008 to record it.  I'll fix all of them.